packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -72,6 +72,11 @@ export class MetricExporter implements IMetricExporter {
       scopes: ['https://www.googleapis.com/auth/cloud-platform'],
     });
 
+    this._monitoring = google.monitoring({
+      version: 'v3',
+      rootUrl: options.rootUrl || 'https://monitoring.googleapis.com/',
```

Comment:
Should we make this the same as trace exporter with `apiEndpoint` and just add the `https://` prefix on our own?

Reply:
It still wouldn't include the port (e.g. the apiEndpoint on trace includes :443 as well).

Reply:
I changed it to add the https:// prefix, and to include the port

Reply:
And I changed it to be called "apiEndpoint" to match the trace client

Reply:
LGTM